File: Week_0.1/q8.c
#include <stdio.h>

int main() {
    int limit, sum = 0;
    printf("Enter a limit : ");
    scanf("%d", &limit);

    for(int i = 1; i <= limit; i += 2) {
        sum += i;
    }

    printf("Sum of odd numbers = %d\n", sum);
}

File: Week_3/q8.js
var readlineSync = require('readline-sync')

let limit = readlineSync.question("Enter a limit ")
const a =[]
let odd = []
let sum

console.log("Enter numbers");
for(let i = 0; i < limit; i++) {
    a[i] = Number(readlineSync.question())
}

odd = a.filter(x => x%2 != 0)
sum = odd.reduce((total, num) => total + num)
console.log(sum);